ideascube/search/migrations/0001_initial.py

```diff
@@ -6,9 +6,10 @@
 
 
 class CreateSearchModel(migrations.CreateModel):
-    def database_forwards(self, *_):
-        # Don't run the parent method, we create the table our own way
-        create_index_table()
+    def database_forwards(self, app_label, schema_editor, from_state, to_state):
+        model = to_state.apps.get_model(app_label, self.name)
+        if self.allow_migrate_model(schema_editor.connection.alias, model):
```

Comment:
Now I wonder... If we monkeypatched `schema_editor.create_model`, then we could use `super().database_forwards` couldn't we?

(no, that would not necessarily be a good idea, but it's been a long day)

Well done in any case here.